contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/tests/rule_based_profiler/data_assistant/test_data_profiler_structured_data_assistant.py

```diff
@@ -64,8 +58,9 @@ def bobby_profile_data_profiler_structured_data_assistant_result_usage_stats_ena
                 "data_profiler_files",
                 "profile.pkl",
             ),
+            "metric": "data_type",
+            "metric_values": ["int", "float"],
```

Comment:
@alexjeilert Please see my note above about possibly changing the name `metric_values` to something more intuitive for this purpose.  Thank you.